### DISCUSSION TITLE
v1.16.0

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#505] [Release]
**Features:**
Add Mod Settings ([Mods#518](https://github.com/R2Northstar/NorthstarMods/pull/518))
Basic mods blacklisting ([Launcher#474](https://github.com/R2Northstar/NorthstarLauncher/pull/474))

**Code-cleanup:**
Reorganize third-party dependencies into `thirdparty` directory ([Launcher#491](https://github.com/R2Northstar/NorthstarLauncher/pull/491), [Launcher#496](https://github.com/R2Northstar/NorthstarLauncher/pull/496))

**Contributors:**
@EladNLG @F1F7Y @GeckoEidechse

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.16.0'>v1.16.0</a>.</em>

**Comment by Ultrafunkalistic** [comment]
I've loaded the v1.16.0 but got 'The version you're using is no longer supported' wuht

**Reply by LumiKattlin** [reply]
Can't you read dates? this is from a year ago,that's not the latest version lmao
